docs/network/smart-rollup-nodes.md

```diff
@@ -22,19 +22,22 @@ You can use a public layer 1 node or set up your own as described in [Installing
 ## Running the Smart Rollup node
 
 You can start the Smart Rollup node with a snapshot of the Etherlink state or from Etherlink genesis.
-These instructions cover starting from Etherlink genesis, which requires a recent build of the Octez suite.
-They result in an archive node, which contains a copy of the state at each level since Etherlink genesis.
 
-For simplicity, these steps show how to run the octez smart rollup node in observer mode:
+- If you start from genesis, the result is an archive node, which contains a copy of the state at each level since Etherlink genesis.
+- If you start from a snapshot, the result is a rolling node, which contains only a copy of the state at the current level.
```

Comment:
Whether to start from a snapshot doesn't actually impact the mode. The snapshot contains the full history